views/class.html

```diff
@@ -2,20 +2,26 @@
   <div class="row">
     <h2>Class <span ng-bind="class_name"></span></h2>
     <hr />
-    <p class="description"><span future-ng-bind="class_description">This is a dummy class!</span></p>
-    <p>Owner: <b><span future-ng-bind="class_owner">mrazo</span></b></p>
+    <p class="description"><span ng-bind="class_description"></span></p>
+    <p>Owner: <b><span ng-bind="class_owner"></span></b></p>
```

Comment:
As far as I know as long as you change the `{{thing}}` within an Angular scope (controller, $http, etc.) it will update. Don't even remember why I used ng-bind tho lol.